client/landing/site.landing/coffee/team/forms/teamdomaintabform.coffee

```diff
@@ -19,42 +19,34 @@ module.exports = class TeamDomainTab extends kd.FormView
       click        : => @input.setFocus()
 
     @inputView.addSubView @input = new kd.InputView
-      placeholder  : 'your-team'
-      attributes   : 10
+      placeholder  : 'Your team name...'
```

Comment:
I used `Your team name...` here by analogy with domain placeholder in `TeamSelectorForm` https://github.com/koding/koding/blob/master/client/landing/site.landing/coffee/teams/teamselectorform.coffee#L24. Previously it was `your-team` there too but I changed it to `Your team name...` according to new design screenshots.
I've fixed both places to have `your-team-name` placeholder as you suggested.